Command/RunParatestCommand.php

```diff
@@ -25,37 +26,26 @@ class RunParatestCommand extends ContainerAwareCommand
     protected function configure()
     {
         $this
-            ->setName('test:run')
+            ->setName('paratest:run')
             ->setDescription('Run phpunit tests with multiple process')
+            ->addArgument('options', InputArgument::OPTIONAL, 'Options')
         ;
     }
 
     protected function prepare()
     {
-        $this->configuration = $this->getContainer()->hasParameter('liip_functional_test');
-        $paratestCfg = (!isset($this->configuration['paratest'])) ? array('process' => $this->process, 'phpunit' => $this->phpunit) : $this->configuration['paratest'];
+        $container = $this->getContainer();
 
-        $this->process = (!empty($this->configuration['process'])) ? $paratestCfg['process'] : $this->process;
-        $this->phpunit = (!empty($this->configuration['phpunit'])) ? $paratestCfg['phpunit'] : $this->phpunit;
-        $this->testDbPath = $this->getContainer()->get('kernel')->getRootDir();
-        $this->output->writeln("Cleaning old dbs in $this->testDbPath ...");
-        $createDirProcess = new Process('mkdir -p '.$this->testDbPath.'/cache/test/');
-        $createDirProcess->run();
-        $cleanProcess = new Process("rm -fr $this->testDbPath/cache/test/dbTest.db $this->testDbPath/cache/test/dbTest*.db*");
-        $cleanProcess->run();
-        $this->output->writeln("Creating Schema in $this->testDbPath ...");
-        $createProcess = new Process('php app/console doctrine:schema:create --env=test');
-        $createProcess->run();
+        $this->process = $container->getParameter('liip_functional_test.paratest.process');
+        $this->paratestPath = $container->getParameter('liip_functional_test.paratest.path');
+        $this->phpunit = $container->getParameter('liip_functional_test.paratest.phpunit');
+        $this->xmlConfig = $container->getParameter('liip_functional_test.paratest.xml_config');
 
-        $this->output->writeln('Initial schema created');
-        $populateProcess = new Process("php app/console doctrine:fixtures:load -n --fixtures $this->testDbPath/../src/overlord/AppBundle/Tests/DataFixtures/ORM/ --env=test");
-        $populateProcess->run();
+        $this->testDbPath = $container->getParameter('kernel.cache_dir');
 
-        $this->output->writeln('Initial schema populated, duplicating....');
-        for ($a = 0; $a < $this->process; ++$a) {
-            $test = new Process("cp $this->testDbPath/cache/test/dbTest.db ".$this->testDbPath."/cache/test/dbTest$a.db");
```

Comment:
But LiipFunctionalTestBundle doesn't use any code like this when we don't use paratest, Doctrine creates the database automatically. I don't understand why it should be different with paratest. Isn't the database created automatically with the path provided in [`createConnection`](https://github.com/liip/LiipFunctionalTestBundle/pull/253/files#diff-83ff5a47a10ec9c63718ab50cad18db4R25)?